calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -73,7 +73,17 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
 
 ### Configure a cluster for WAF
 
-#### Step 1: Configure ApplicationLayer CRD
+#### Step 1: Enable WAF in your cluster
+
+On the Manager UI, click **Threat Defense -> Web Application Firewall -> Configure Web Application Firewall**.
+
+<img
```

Comment:
I think the docs usually don't include screenshots, unless the screen is complicated enough that text describing it isn't enough. See this slack [message](https://tigera.slack.com/archives/C02BRQ66W8P/p1674501295758419?thread_ts=1674492642.842479&cid=C02BRQ66W8P) from @malavikabala

Reply:
I have removed the screenshot.